# First and second order formation statistics

std_msgs/Header header

# ids of the neighbours of the current agent (the ones it communicates with)
int32[] neighbours_
# vector of the agent estimated statistics
FormationStatisticsStamped[] vector
